README.md

```diff
@@ -92,6 +101,25 @@ mods.
 * committer shall due their due dilligence cooperating with other committer, author and
 admin in maintaining the mod as a team.
 
+## Governance
+
+The organization shall regularly assess and verify that enough administrators are active
+and acting in the duty of administrator.
+
+The organization and it's administrators shall periodically report on and consult with
+authors and committers to determine if the amount of administrators is sufficient
+for the proper functioning of the organization. If the current amount of administrators
+is found to be insufficient by either the admins or the authors, then the organization
+including admins, authors and committers shall appoint the appropriate amount of
+new admins to the organization, and the existing admins shall grant the adminstrator
+access to the new admin on the organization's github organization.
```

Comment:
I don't like the emphasis on "number".  You can't just count people to determine how capable they are of handling a workload.  I think this should focus more on whether the current administrators are able to handle the workload.